Cargo.toml

```diff
@@ -1,15 +1,17 @@
 [package]
-name = "send_wrapper"
+name = "parity-send-wrapper"
 version = "0.2.0"
```

Comment:
Increase version? Or set back to `0.1.0`?

Reply:
I guess I'll set to 0.1.0!